recipe/build.sh

```diff
@@ -0,0 +1,16 @@
+# we need to call cmake first because the `python -m pip install` step does not build dynamic libraries
+# and does not include headers
+cmake -DCMAKE_BUILD_TYPE=Release \
+      -DPYTHON_EXECUTABLE=${PREFIX}/bin/python \
+      -DKALDI_NATIVE_FBANK_BUILD_TESTS=ON \
+      -DKALDI_NATIVE_FBANK_BUILD_PYTHON=ON \
+      -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+      -GNinja -B build
+
+cmake --build build --target install
+
+# create python bindings
+$PYTHON -m pip install . -v --no-deps --no-build-isolation
```

Comment:
WOn't this trigger a second recompilation due to https://github.com/csukuangfj/kaldi-native-fbank/blob/v1.17/cmake/cmake_extension.py#L53-L120?

Reply:
it will, that's why I do `https://github.com/AnacondaRecipes/kaldi-native-fbank-feedstock/blob/923675ef2a58a512a2e52152162952baad8695b2/recipe/build.sh#L16` immediately after. I could patch out the second build in theory.

Reply:
if I don't run `cmake` manually only `$PYTHON -m pip install . -v --no-deps --no-build-isolation` is run (I checked). In this case, however, the test binaries are not built. I wanted to have the flexibility of being able to test the package as well as building the python bindings, after all this package is a python package.

Reply:
My question still stands. DO you really need the python bindings?

Reply:
not for fairseq2, no.

Reply:
So in this case, I would recommend to not build the python binding.

Reply:
ok.